app/src/org/commcare/network/HttpRequestGenerator.java

```diff
@@ -168,6 +169,12 @@ public HttpResponse makeCaseFetchRequest(String baseUri, boolean includeStateFla
         //Add items count to fetch request
         serverUri = serverUri.buildUpon().appendQueryParameter("items", "true").build();
 
+        if (CommCareApplication._().shouldInvalidateCacheOnRestore()) {
+            serverUri = serverUri.buildUpon().appendQueryParameter("overwrite_cache", "true").build();

```

Comment:
I had figured it was sort of apparent what it does from the name of the parameter. Should I add something that says it's currently only used for testing purposes?